.github/workflows/update_dependencies_and_client.yml

```diff
@@ -94,3 +94,18 @@ jobs:
             firease_api
           reviewers: bootstraponline,jan-gogo,pawelpasterz,adamfilipow92,piotradamczyk5,Sloox,axelzuziak-gogo
           draft: false
+
+  check_for_sdk_updates:
+    runs-on: macos-latest
+    needs: [ get_token_and_date ]
```

Comment:
You are using `secrets.GITHUB_TOKEN ` so I do not think that you need to wait for `get_token_and_date` job

Reply:
Good point. 
BTW, do you think I could remove `create gh token` at all? It's used 
https://github.com/Flank/flank/blob/d1814271875824d7c4ae286566a0637a9e026ea6/.github/workflows/update_dependencies_and_client.yml#L49
and here 
https://github.com/Flank/flank/blob/d1814271875824d7c4ae286566a0637a9e026ea6/.github/workflows/update_dependencies_and_client.yml#L86
and can be replaced with `secrets.GITHUB_TOKEN` ?

Reply:
no you could not use `secrets.GITHUB_TOKEN` in those cases.
The reason why we are using a custom token (from Github App) is that we create PR with changes (default Github token can not trigger workflows)

Reply:
I see, thanks for explanation